packages/astro/test/fixtures/lit-element/src/components/my-element.js

```diff
@@ -3,9 +3,26 @@ import { LitElement, html } from 'lit';
 export const tagName = 'my-element';
 
 export class MyElement extends LitElement {
+  static properties = {
+    bool: {type: Boolean},
+    str: {type: String},
```

Comment:
since `String` is the default type, I'd make sure this one is working by giving it a different attribute name, like `attribute: str-attr`.

Reply:
btw it'll still leave the attribute on the host since it doesn't pass the prototype check. Might prove to be problematic with a different attr `type:Boolean`, but this is better than before